docs/operating-systems/course-tools.md

```diff
@@ -148,51 +143,52 @@ If you haven’t installed gatorgrade type this command:
 
 ### Go installation process:
 
-If you haven't installed go click the link and follow [the instructions](https://go.dev/dl/)
+If you haven't installed Go: [the instructions](https://go.dev/dl/)
 
 Make sure you click the right chip that your laptop has.
 
 If you choose the right version and it still isn't working, follow these instructions:
-1. type in the command ```nano ~/.zshrc``` into your terminal
-2. add in the following 2 paths into the file that pops up in your terminal:
+- Type in the command ```nano ~/.zshrc``` into your terminal
+- Add in the following 2 paths into the file that pops up in your terminal:
   - ```export PATH=$PATH:/usr/local/go/bin```
   - ```export PATH=$PATH:$GOPATH/bin```
-3. After you save the changes in that file, source the file using the command ```. ~/.zshrc```.
-4. Check if go is now working using the ```go version``` command.
+- After you save the changes in that file, source the file using the command ```. ~/.zshrc```.
+- heck if go is now working using the ```go version``` command.
 
 
 ### GCC installation process:
 
-GCC might already be installed on your computer since you have a mac. To check type:
+GCC might already be installed on your computer since you have a Mac. To check type:
 
 `gcc –version`
```

Comment:
This command is incorrect, it should be gcc --version, it is missing one of the required dashes